buggy_utils.py

```
@@ -0,0 +1,107 @@
+import os
+import subprocess
+
+# SQL Injection vulnerability
+def get_user(username):
+    import sqlite3
+    conn = sqlite3.connect("users.db")
+    query = "SELECT * FROM users WHERE name = '" + username + "'"
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)